### ISSUE TITLE
Login vs Re-login using SDDM in Manjaro KDE

### ISSUE THREAD

**Issue by cprpolo** [sddm/sddm#1683]
Hi all,I use manjaro and Plasma KDE

OS: Manjaro Linux x86_64  
Host: MS-7821 1.0   
Kernel: 6.1.12-1-MANJARO   
DE: Plasma 5.26.5   
WM: KWin   
Theme: Breath2 2021 Light [Plasma], Breath [GTK2/3]   
Icons: breeze [Plasma], breeze [GTK2/3]  

I am using SDDM for login screen and I have changed the default theme to: Sugar Candy (https://framagit.org/MarianArlt/sddm-sugar-candy) and it works on first login but when locking the screen the login screen (to re-login) is still the default theme (Manjaro Team). I have tried to figure it out but I am not capable...

Is this expected? if so, how do I get Sugar Candy in the "relogin" screens after locking the PC?

Thank you in advance :)

Adding after some research:
**/etc/sddm.conf:**
```
[General]InputMethod=Namespaces=Numlock=on

[Theme]DisableAvatarsThreshold=7EnableAvatars=trueFacesDir=/usr/share/sddm/facesThemeDir=/usr/share/sddm/themesCurrent=sugar-candy[Users]DefaultPath=/usr/local/sbin:/usr/local/bin:/usr/binHideShells=HideUsers=RememberLastSession=trueRememberLastUser=trueReuseSession=true

[Wayland]EnableHiDPI=trueSessionCommand=/usr/share/sddm/scripts/wayland-sessionSessionDir=/usr/share/wayland-sessionsSessionLogFile=.local/share/sddm/wayland-session.log

[X11]DisplayCommand=/usr/share/sddm/scripts/XsetupDisplayStopCommand=/usr/share/sddm/scripts/XstopEnableHiDPI=trueMinimumVT=1ServerArguments=-nolisten tcpServerPath=/usr/bin/XSessionCommand=/usr/share/sddm/scripts/XsessionSessionDir=/usr/share/xsessionsSessionLogFile=.local/share/sddm/xorg-session.logUserAuthFile=.XauthorityXauthPath=/usr/bin/xauthXephyrPath=/usr/bin/Xephyr

```

**/etc/sddm.conf.d/00_manjaro_settings.conf**

```[Theme]Current=sugar-candy ```

**/etc/sddm.conf.d/kde_settings.conf**
```
[Autologin]Relogin=false#Session=#User=

[General]HaltCommand=/usr/bin/systemctl poweroffRebootCommand=/usr/bin/systemctl reboot

[Theme]Current=sugar-candyCursorTheme=Bibata-Modern-IceFont=Noto Sans,10,-1,0,50,0,0,0,0,0

[Users]MaximumUid=60513MinimumUid=1000 ```

**Comment by Yuri-K7** [comment]
If you only lock the screen, you're still in your DE's session, not sddm. To return to sddm you need to log out of your session.

Search "Screen locking" in your settings.

**Comment by cprpolo** [comment]
Hi @Yuri-K7  thanks for the reply
In Screen Locking there is nothing related to login options or theme, it's only wallpaper:

![image](https://user-images.githubusercontent.com/15221926/224671386-ae902553-001e-436f-85d4-3cd79f4c0cf3.png)

is that where you are referring to?

**Comment by Yuri-K7** [comment]
@cprpolo yes, sorry I'm just a user like you, I only know that way to customize it.

**Comment by cprpolo** [comment]
@Yuri-K7 thank you, maybe we can get someone to assist... :)
